src/main/resources/i18n/messages_ru.properties

```diff
@@ -576,4 +576,6 @@ mapVault.ladder = Рейтинговые карты
 news.showLadderMaps = Показать рейтинговые карты
 ranked1v1.showMaps = Показать рейтинговые карты
 settings.language = Язык
-settings.notifications.displayNotification = Показывать уведомление
\ No newline at end of file
+settings.notifications.displayNotification = Показывать уведомление
```

Comment:
Never do translation here but on poeditor

Reply:
Got into the setting in the client and then language and u will find a link to it

Reply:
but why? I made a correct translation into Russian

Reply:
because we have an app for translations and if you dont enter it there it gets overwritten